rising/transforms/functional/channel.py

```diff
@@ -6,25 +6,34 @@
 __all__ = ["one_hot_batch"]
 
 
-def one_hot_batch(target: torch.Tensor, num_classes: Optional[int] = None) -> torch.Tensor:
+def one_hot_batch(target: torch.Tensor,
+                  num_classes: Optional[int] = None,
+                  dtype: Optional[torch.dtype] = None) -> torch.Tensor:
```

Comment:
If we add support for passing a specific dtype here, we also need to add support for passing devices, I think. Either both or nothing :D

Reply:
I added the dtype because the output is always of type long. When we think of segmentation pipelines the onehot encoding should be of type float for the rest of the transformations. If you prefer, we can also add the device here